README.md

```diff
@@ -1,2 +1,37 @@
-# portfolio
-My personal portfolio with my most relevant projects.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> My personal portfolio with my most relevant projects.
+
+Here you will eventually find some awesome projects built by yours truly.
+
```

Comment:
Good job setting up this README file.
If check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.

kindly add the following;
- screenshot from your app.
- live demo link
- setup

Example of a good setup that helps the non-technical user to set up your app easily below;
```
Example: 
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- open `index.html` with live server
```